Add js step to TX yamls for subject truncation; add chrome flag to PA Gov's yaml

OdeclasV opened the pull request:
Also deleting a few TX yamls that are no longer active

drewsposeep commented on members/TXL000217.yaml:
```diff
@@ -45,6 +45,18 @@ contact_form:
+          values: [ "$SUBJECT" ]
+          commands: [ "elements[0].value = elements[0].value.length > 44 ? elements[0].value.substring(0,41) + '...' : elements[0].value" ]
+          required: true
+    - javascript:
```
repeated

OdeclasV replied:
good catch

drewsposeep commented on members/TXL000218.yaml:
```diff
@@ -45,6 +45,18 @@ contact_form:
+          values: [ "$SUBJECT" ]
+          commands: [ "elements[0].value = elements[0].value.length > 44 ? elements[0].value.substring(0,41) + '...' : elements[0].value" ]
+          required: true
+    - javascript:
```
repeated

OdeclasV replied:
good catch

drewsposeep commented on members/state_pa_gov_tom_wolf.yaml:
```diff
@@ -1,14 +1,17 @@
-      - selector: "#newsletterModal2 button"
+      - selector: "#nf-form-421-cont"
+    - find:
+      - selector: "#nf-field-3191"
```
do we need both of these?

OdeclasV replied:
I took the first `find` out and left the one related to the `firstname` selector

tirish commented on members/TXL000195.yaml:
```diff
@@ -53,6 +53,12 @@ contact_form:
+    - javascript:
+        - name: subject truncation
+          selectors: [ "form#memberForm input#subject"]
+          values: [ "$SUBJECT" ]
```
You can leave out `values` entirely, since there is no reference to it in the command

OdeclasV replied:
updated

tirish commented on members/TXL000195.yaml:
```diff
@@ -53,6 +53,12 @@ contact_form:
           required: true
+    - javascript:
+        - name: subject truncation
+          selectors: [ "form#memberForm input#subject"]
```
Can this selector simply be `#subject`?

OdeclasV replied:
updated

OdeclasV commented:
I took `values` out, per Tommy's suggestion, for all TX yamls. 

In addition, I did more updates for TX Senate targets:

- Simplified the selectors
- Added missing phone variables
- Update the max length in the JS steps to 50 chars

tirish commented on members/STXLc4e68274.yaml:
```diff
@@ -45,6 +45,11 @@ contact_form:
+    - javascript:
+        - name: subject truncation
+          selectors: [ "input[name='subject']"]
+          commands: [ "elements[0].value = elements[0].value.length > 44 ? elements[0].value.substring(0,41) + '...' : elements[0].value" ]
```
Should this be 50?
```suggestion
          commands: [ "elements[0].value = elements[0].value.length > 50 ? elements[0].value.substring(0,47) + '...' : elements[0].value" ]
```

tirish commented on members/TXL000214.yaml:
```diff
@@ -45,6 +45,11 @@ contact_form:
+    - javascript:
+        - name: subject truncation
+          selectors: [ "input[name='subject']"]
+          commands: [ "elements[0].value = elements[0].value.length > 44 ? elements[0].value.substring(0,41) + '...' : elements[0].value" ]
```
50?

OdeclasV replied:
It's 44 chars for TX Reps (targets with URLs like this `http://www.house.state.tx.us...`): https://gyazo.com/9398a673439e6b66316060df29d8b651

And 50 chars for TX Sens (targets with URLs like this `http://www.senate.state.tx...`): https://gyazo.com/366825fd59b54ed6af8a5865e010af5c

tirish replied:
ah fun

tirish reviewed (commented):
I did ctrl+f for "44" and looks like those didn't get set to 50. Was that 44 -> 50 change not for all of them?
